english/super-man/save-the-word.md

```diff
@@ -0,0 +1,12 @@
+After that, suddenly!!! You start to grow up, Big, GIANT ~~~ Also you noticed that a red shirt is now wrapped on your muscular body, and the trangle underwear is mounted on it, shining a great "V" -- the signature of SUPER MAN.
+
+As the most glory superman, you decide to save the world. Do you:
+
+[Cathes the robbers in the UN bank?]
```

Comment:
Please don't add all these options that don't link anywhere. Otherwise, the story will soon become full of options you can't choose, which is frustrating for a reader. I'm sure soon someone will add options to this page, so for now you could just remove them.